ext/zip/config.m4

```diff
@@ -85,7 +85,7 @@ if test "$PHP_ZIP" != "no"; then
 
     if test -z "$LIBZIP_LIBDIR"; then
       AC_MSG_RESULT(not found)
-      AC_MSG_ERROR(Please reinstall the libzip distribution)
+      AC_MSG_ERROR(Please install the required header files for libzip, this may be libzip-dev)
```

Comment:
debian specific, IMHO no value

Reply:
```suggestion
      AC_MSG_ERROR(Please install the required header files for libzip. Header zip.h not found.)
```

Then let's add this one maybe? I think there were similar issues also with some other cases on bugs.php.net where only header file(s) for libzip were missing (meaning, the development package wasn't installed).
cc @7snovic